Automattic/wp-calypso
# Blaze header hidden by default + new translation strings added

Related to 1448-gh-Automattic/dotcom-forge

<img width="1320" alt="image" src="https://user-images.githubusercontent.com/1044309/223536098-7ba753d3-7f21-4710-87e2-2471a48d0a37.png">


## Proposed Changes

* Hidden blaze header by default. This is necessary to not flicker the screen when loading Blaze campaign creator (DSP-client)
* Added new translation strings, here more context: p1677795894075239-slack-C0351BKR0M6

## Testing Instructions



* [Apply this Tumblr PR](https://github.tumblr.net/Tumblr/wordads-picard/pull/1667) in your local dsp-client. Here more information on how to do that: pb5gDS-2Iw-p2 
* Open /advertising in your Calypso and Promote a new post
* Make sure you won't see Calypso header until dismiss the WelcomeMessage
* Make sure the strings match with [the original Tumblr PR](https://github.tumblr.net/Tumblr/wordads-picard/pull/1648)



Before | After
--|--
![image](https://user-images.githubusercontent.com/1044309/223578086-2e80b1ab-994e-4f45-a5a0-4e2e74af0f9c.png)  |  ![image](https://user-images.githubusercontent.com/1044309/223578103-896a487d-347c-4ede-8be7-4ac7b687f992.png)









## Pre-merge Checklist
